www/src/Handlers/Admin.php

```diff
@@ -83,6 +83,14 @@ public static function cacheCheck(RequestContext $request_context): Response
 
     public static function getTestInfo(RequestContext $request_context): Response
     {
+
+        // psalm workaround
+        if (!function_exists('GetTestInfo')) {
```

Comment:
oh man, this is amazing. good for keeping stock of all the stuff we need to move, too